### DISCUSSION TITLE
mailcap file compatibility

### DISCUSSION THREAD

**Discussion by DNied** [zim-desktop-wiki/zim-desktop-wiki#1956] [Ideas]
Hi! I would like Zim to be able to use the `mailcap` file mechanism for associating MIME types to applications.

Like probably many other command-line users, I have set it up with lots of entries, through years of evolving needs and changing software. Needing to once again associate all apps and MIME types via the XDG tools and formats (which I don't use) is frankly a chore I would happily avoid.

**Comment by jaap-karssenberg** [comment]
Have a look at patching the "zim.applications" module. Current behavior is
to check for xdg and fallback to the python mimetypes library if not found.
You might add a similar fallback for the mailcap library.
